Packs/Arcanna/pack_metadata.json

```diff
@@ -10,10 +10,7 @@
     "categories": [
         "Data Enrichment & Threat Intelligence"
     ],
-    "tags": [
-        "Security",
-        "Machine Learning"
-    ],
```

Comment:
Here it should be "Decision Intelligence"

Reply:
Hey Marian,
we are reviewing and updating the list of approved tags, your Partner Success Engineer will contact you regarding the possibilities in the near future. First, we are aligning the XSOAR and XSIAM Marketplaces to the approved baseline.